topic-controller/src/test/java/io/strimzi/test/RoleBinding.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.strimzi.test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Describes a Kubernetes RBAC {@code RoleBinding}.
+ * @see <a href="https://kubernetes.io/docs/admin/authorization/rbac">Kubernetes RBAC docs</a>
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface RoleBinding {
+    /** The name of the role binding. If not specified it will default to the Role name + "-binding". */
```

Comment:
Do we need to sync this with the YAML files / YAML files with this? Cluster Controller YAML files are `strimzi-cluster-controller-role` and ` strimzi-cluster-controller-binding` ... so adding `-binding` to the role name would not work. But I'm not sure how this is related to the YAML files right now.

Reply:
Yeah, I'm not sure this bit of laziness is really worthwhile. We should make the role binding name mandatory in the annotation.